ubuntudesign/tutorials.ubuntu.com
# Modify tutorial loading

Modify loading of tutorials content.

- Remove duplicated styles across tutorials
- Remove iron-pages nesting
- This PR should solve #47 

### QA 
- Ensure tutorials load.
- Make sure images load.
- Verify styling of survey and codeblocks matches current style
